packages/neon-core/src/sc/InteropServicePrices.ts

```diff
@@ -0,0 +1,125 @@
+  [InteropServiceCode.SYSTEM_HEADER_GETINDEX]: 400e-8,
+  [InteropServiceCode.SYSTEM_HEADER_GETHASH]: 400e-8,
+  [InteropServiceCode.SYSTEM_HEADER_GETPREVHASH]: 400e-8,
+  [InteropServiceCode.SYSTEM_HEADER_GETTIMESTAMP]: 400e-8,
+  [InteropServiceCode.SYSTEM_BLOCK_GETTRANSACTIONCOUNT]: 400e-8,
+  [InteropServiceCode.SYSTEM_BLOCK_GETTRANSACTIONS]: 10000e-8,
+  [InteropServiceCode.SYSTEM_BLOCK_GETTRANSACTION]: 400e-8,
+  [InteropServiceCode.SYSTEM_TRANSACTION_GETHASH]: 400e-8,
+  [InteropServiceCode.SYSTEM_CONTRACT_CALL]: 1000000e-8,
+  [InteropServiceCode.SYSTEM_CONTRACT_DESTROY]: 1000000e-8,
+  [InteropServiceCode.SYSTEM_STORAGE_GETCONTEXT]: 400e-8,
+  [InteropServiceCode.SYSTEM_STORAGE_GETREADONLYCONTEXT]: 400e-8,
+  [InteropServiceCode.SYSTEM_STORAGE_GET]: 1000000e-8,
+  [InteropServiceCode.SYSTEM_STORAGE_PUT]: undefined,
+  [InteropServiceCode.SYSTEM_STORAGE_PUTEX]: undefined,
+  [InteropServiceCode.SYSTEM_STORAGE_DELETE]: 1000000e-8,
+  [InteropServiceCode.SYSTEM_STORAGECONTEXT_ASREADONLY]: 400e-8,
+  [InteropServiceCode.NEO_NATIVE_DEPLOY]: 0e-8,
+  [InteropServiceCode.NEO_CRYPTO_CHECKSIG]: 1000000e-8,
+  [InteropServiceCode.NEO_CRYPTO_CHECKMULTISIG]: undefined,
+  [InteropServiceCode.NEO_HEADER_GETVERSION]: 400e-8,
+  [InteropServiceCode.NEO_HEADER_GETMERKLEROOT]: 400e-8,
+  [InteropServiceCode.NEO_HEADER_GETNEXTCONSENSUS]: 400e-8,
+  [InteropServiceCode.NEO_TRANSACTION_GETSCRIPT]: 400e-8,
+  [InteropServiceCode.NEO_TRANSACTION_GETWITNESSES]: 10000e-8,
+  [InteropServiceCode.NEO_WITNESS_GETVERIFICATIONSCRIPT]: 400e-8,
+  [InteropServiceCode.NEO_ACCOUNT_ISSTANDARD]: 30000e-8,
+  [InteropServiceCode.NEO_CONTRACT_CREATE]: undefined,
+  [InteropServiceCode.NEO_CONTRACT_UPDATE]: undefined,
+  [InteropServiceCode.NEO_CONTRACT_GETSCRIPT]: 400e-8,
+  [InteropServiceCode.NEO_CONTRACT_ISPAYABLE]: 400e-8,
+  [InteropServiceCode.NEO_STORAGE_FIND]: 1000000e-8,
+  [InteropServiceCode.NEO_ENUMERATOR_CREATE]: 400e-8,
+  [InteropServiceCode.NEO_ENUMERATOR_NEXT]: 1000000e-8,
+  [InteropServiceCode.NEO_ENUMERATOR_VALUE]: 400e-8,
+  [InteropServiceCode.NEO_ENUMERATOR_CONCAT]: 400e-8,
+  [InteropServiceCode.NEO_ITERATOR_CREATE]: 400e-8,
+  [InteropServiceCode.NEO_ITERATOR_KEY]: 400e-8,
+  [InteropServiceCode.NEO_ITERATOR_KEYS]: 400e-8,
+  [InteropServiceCode.NEO_ITERATOR_VALUES]: 400e-8,
+  [InteropServiceCode.NEO_ITERATOR_CONCAT]: 400e-8,
+  [InteropServiceCode.NEO_JSON_SERIALIZE]: 100000e-8,
+  [InteropServiceCode.NEO_JSON_DESERIALIZE]: 500000e-8,
+  [InteropServiceCode.NEO_NATIVE_POLICY]: undefined,
+  [InteropServiceCode.NEO_NATIVE_TOKENS_NEO]: undefined,
+  [InteropServiceCode.NEO_NATIVE_TOKENS_GAS]: undefined
+};
+
+function getStoragePrice(size: number): number {
+  return size * GAS_PER_BYTE;
+}
+
+function getCheckMultiSigPrice(size: number): number {
+  return size * fixedPrices[InteropServiceCode.NEO_CRYPTO_CHECKSIG];
+}
+
+export function getInteropServicePrice(
+  service: InteropServiceCode,
+  param?: Partial<InteropServicePriceParam>
```

Comment:
Well this would still let user to "select" the already-decided method from a list.
How about this:
```js
export const InteropService = {
    NEO_ACCOUNT_ISSTANDARD = generateGeneralService('aaaaaa'),
    NEO_CRYPTO_CHECKMULTISIG =generateMultisigService('bbbbbb')
}

function generateGeneralService(code: string) {
  return {
    code,
    getPrice: () => new Fixed8()
  }
}

function generateMultisigService(code: string) {
  return {
    code,
    getPrice: (size: number) => new Fixed8(size * FACTOR)
  }
}
```

Reply:
There isnt a point in returning the code? I think the easier thing would be to provide a map of code to functions.

Reply:
> There isnt a point in returning the code?

The above code means to combine `InteropServiceCode` & `InteropServiceCodePrices`
> Either way, the call will become something like calculatePriceForCode(InteropServiceCode code, int param).

The name `param` is too vague, and it can have 2 meanings in actual use.

Reply:
Lets leave the code enum as it is. I think the code will look a little weird like `InteropService[NEO_CRYPTO_CHECKMULTISIG].code`. And where do I look up all the strings?